### PR TITLE
Iain

### PR DESCRIPTION
**Pull request by iainardo**
Hi @marc-ed-raffalli, I have created a PR as we discussed:
- I've made some changes to fix typos
- I've added a suggested outline for the electronics camp
 
Please let me know what you think. Thanks.

### CODE REVIEW

File: pages/en/partials/class-electronic.md [hunk 1]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
 ---
 
 Electronic class
+
+The following provides information about the Athlone Coder Dojo electronics camp
+
+## Objective
+
+The objective is teach basic concepts of electronics to kids, learing physical computing in a fun informal way.
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: `learing ` => lear**n**ing (typo)

File: pages/en/partials/class-electronic.md [hunk 2]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+- Understand basic electronic components (e.g. a resistor, capacitor, diode, ..)
+    - Distinguish between active and passive components and components polarity
+- Understand how to read a basic electronic circuit diagram
+- Creating basic circuits virtually using software tools
+- Build circuits in teams using a breadboard, jumper cables and a variety of components
+- Understanding voltage / current / resistance and their unit of measurement
+```
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: You can replace this block with:
```
    {% include alert.html
    type="danger"
    content="Those who attend are constantly reminded the dangers of experimenting with electronics
                     themselves without a full understanding of the possible consequences!"
    %}
```

File: pages/en/partials/class-electronic.md [hunk 3]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+- Learn about microcontrollers (Arduino)
+- Learn to program an Arduino
+   * Use "sketch" to create programs
+   * Perform various hands on experiments using an Arduino
+- Create various projects using Arduino (following Dojo projects)
+- Conduct various mini projects throughout the year (2-3 weeks)
+- Introduce the Raspberry Pi
+   * Adventure through the world of physical computing with a Pi!
+- Work towards coming up with own project ideas and building a project as a team which could be entered into the Coder Dojo coolest project competition!
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: > Work towards coming up with own project

Is there a work missing?

File: pages/en/partials/class-electronic.md [hunk 4]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+
+## Pre-requisites
+- It is recommended that the course be undertaken by children 12 years and older
+- It is assumed kids will have previously attended Scratch/Web CoderDojo sessions
+- Have a willingness to learn and have fun :-)
+
+## When
+- The course is run during school term every Saturday from 11am-1pm.
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: This is in the `en/about/class-information.html` :)

File: pages/en/partials/class-electronic.md [hunk 5]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+- Any Saturdays when the class isn't running will be announced in advance.
+
+## Labs
+Various hands on labs will be conducted throughout the year including:
+- Use "Hot wires" snap circuits to build circuits
+- Take apart a computer - understand various internal parts
+- Brining in old toys with motors
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: `Brining` => typo?

File: pages/en/partials/class-electronic.md [hunk 6]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+## References / Resources
+
+* [Arduino resources](http://kata.coderdojo.com/wiki/Arduino_Path) Dojo Kata
+* [Raspberry Pi resources](https://curriculum.raspberrypi.org/physical-computing/creator/)
+* Coolect Projects [FAQ](https://help.coolestprojects.org/hc)
+* [Coder Dojo resources](https://coderdojo.com/resources/)
+
+## House rules
+
+- Kids under 13 must be accompanied by an adult
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: This is great, can you check the page `en/about/class-information.html` and put the list there?

File: pages/en/partials/class-electronic.md [hunk 7]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+- Kids are welcome to bring along their own electronics kits, if they have them.
+  - Breadboard, Resistors, LEDs
+  - Arduino , Raspberry Pi
+- Mentors will offer recommendations on kits to purchase if parents wish to do this.
+- The group mentors have a limited supply of electronic equipment which the group can use / share
+- For those kids who bring their own kits, they take responsibility for looking after their equipment.
+
+## Other things to note
+
+- If a quota of volunteers are not available in any particular week the course may not run or will run in a limited capacity
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md:99]: Same here, good thoughts applicable to all classes

File: pages/en/partials/class-electronic.md [hunk 8]
```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
...
+. At least two mentors must be available for a weekly session proceed. A single mentor cannot take a whole class in a classroom by themselves for safety reasons.
```
Comment (marc-ed-raffalli) [inline pages/en/partials/class-electronic.md]: `. At least...` was that meant to be a dash? :)

File: pages/en/partials/class-electronic.md (hunk 1 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md]: Fixed

File: pages/en/partials/class-electronic.md (hunk 5 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md]: Indeed. Fixed

File: pages/en/partials/class-electronic.md (hunk 6 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md]: It seems like the page `en/about/class-information.html` already has this content, so removing from `pages/en/partials/class-electronic.md` as per your request

File: pages/en/partials/class-electronic.md (hunk 3 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md]: Reworded

File: pages/en/partials/class-electronic.md (hunk 4 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md]: Removed

File: pages/en/partials/class-electronic.md (hunk 7 above)
Comment (iainardo) [reply pages/en/partials/class-electronic.md:99]: I suggest to leave this statement here as it is specific to the electronics group, as there may weeks where no mentors are available so the electronics dojo may not run at all.

Comment (marc-ed-raffalli) [reply pages/en/partials/class-electronic.md:99]: ok

**Review by marc-ed-raffalli — APPROVED** [review]

**Pull request merged**